packages/react/src/app-layout/AppLayout.tsx

```diff
@@ -9,9 +9,15 @@ import {
 export type AppLayoutProps = PropsWithChildren<{
 	/** Set to `true` while users are completing multi-page forms to reduce distractions. When true, the app layout sidebar will not be rendered. */
 	focusMode?: boolean;
+	/** Controls what colour theme the AppLayout components display in, `'bodyAlt'` is light theme. */
+	pallet?: AppLayoutPallet;
```

Comment:
I'm not sure if I should go with `pallet: light/dark` or `background: body/bodyAlt`

Both are existing patterns in the design system but both have problems.

## background: body/bodyAlt
This is the more frequently used pattern on the site. 

**pros**
- The more common pattern
- Makes it easy to handle dark mode later

**cons**
- `body` vs `bodyAlt` gives no indication of which one leads to a lighter colour theme.
- `background` is not a very appropriate word for handling theming at such a high level. It is not changing just one background, it's changing multiple backgrounds.

## pallet: light/dark

**pros**

- better word for reflecting the fact that this is handling a theme change, not just a background
- "light" and "dark" give clear indication of which theme is the light theme and which is the darker theme.

**cons**

- This is the less common pattern on the site
- Since one of the values is "dark", it is going to make introducing a dark mode more difficult later on